chingu-voyages/v52-tier3-team-38
# Gary/feature/mapview

# Description
This PR configures the map view for the application.

## Related to the following Jira/Trello issue 
https://trello.com/c/P9MUxllH/33-view-resident-requests-map-view

## Main changes explained:
- Adjusted appointment map view to place markers showing all locations of requested appointments.
- Clicking on a marker shows more info about the appointment such as the users name, address, time slot, and contact info.

## How to test:
1. check into current branch
2. do `npm install` and `npm run dev` to run this PR locally
3. Login as admin and toggle over to the map view. Verify that markers are present and clicking on one shows more details about the selected appointment.

##Note:
The map view only shows appointments that are pending. Visited appointments will not be shown.
